new-site/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "hds-site",
+  "private": true,
+  "description": "Documentation for Helsinki Design System",
+  "version": "0.1.0",
+  "dependencies": {
+    "gatsby": "^3.13.1",
+    "gatsby-plugin-image": "^1.13.0",
+    "gatsby-plugin-manifest": "^3.13.0",
+    "gatsby-plugin-offline": "^4.13.0",
+    "gatsby-plugin-react-helmet": "^4.13.0",
+    "gatsby-plugin-sharp": "^3.13.0",
+    "gatsby-source-filesystem": "^3.13.0",
+    "gatsby-transformer-sharp": "^3.13.0",
+    "hds-core": "^1.5.0",
```

Comment:
Would it be possible to use hds-core and hds-react directly from the repository instead of as a dependency? 

One possible benefit would be that then the site is always aligned with the newest versions of core and react. No need to update these version numbers. Developing the site would be faster, since you don't have to release hds-core and hds-react to get the new component version. But you can just do the modifications to the same pr. I know we could use `yarn link` to link local versions, but it has some drawbacks: kind of slow, and also you need to switch it off by running yarn unlink.

One possible drawback could be that then the way we use those components from core and react in this site would not be similar of how they are used by consumer projects of HDS. The bundled versions of HDS might introduce a bug that would not show itself if we just directly import the component. 🤔 

Slightly related, Im not sure which option would be better for the feature of supporting multiple versions of HDS documentation in the site.

Reply:
I'm not sure how does it work in build step?

Reply:
Yeah it might be technically harder. We could think about it later 👍

Reply:
> One possible benefit would be that then the site is always aligned with the newest versions of core and react. No need to update these version numbers. Developing the site would be faster, since you don't have to release hds-core and hds-react to get the new component version.
As a document writer, I would love this feature so I could easily create documentation in the feature branch.